htdocs/install/mysql/tables/llx_product_customer_price_extrafields.key.sql

```diff
@@ -0,0 +1,21 @@
+-- ===================================================================
+-- Copyright (C) 2011 Laurent Destailleur       <eldy@users.sourceforge.net>
+-- Copyright (C) 2024		Mélina Joum				<melina.joum@altairis.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <http://www.gnu.org/licenses/>.
+--
+-- ===================================================================
+
+
+ALTER TABLE llx_product_customer_price_extrafields ADD UNIQUE INDEX uk_product_customer_price_extrafields (fk_object);
```

Comment:
To follow contributions rules, thanks to push first the database addition into a separated PR ?